### DISCUSSION TITLE
SVM Classifier

### DISCUSSION THREAD

**Discussion by JoshitaReddy** [JoshitaReddy/TARP---Team-Up#5] [Show and tell]
Hey guys
I have used a SVM classifier for predicting the career.
Support vector machines (SVMs) are a set of supervised learning methods used for classification and regression as well.
![image](https://user-images.githubusercontent.com/78225646/229368979-376134f2-bfd1-45ba-9fd1-09c13aedba84.png)

Support vector machines are:
- Effective in high dimensional spaces.
- Still effective in cases where number of dimensions is greater than the number of samples.
- Uses a subset of training points in the decision function (called support vectors), so it is also memory efficient.
- Versatile: different kernel functions can be specified for the decision function. Common kernels are provided, but it is also possible to specify custom kernels.

Do have a look at the link below for more information!
https://www.javatpoint.com/machine-learning-support-vector-machine-algorithm

**Comment by KeerthikaReddy02** [comment]
Nice! Please push the code for the same to this repo if you have deployed the model.

**Reply by JoshitaReddy** [reply]
Done!

**Comment by SaiRuthvik12** [comment]
That's great, have you checked the model accuracy? If so, then what method have you used and what is the accuracy value?

**Reply by JoshitaReddy** [reply]
I got 86.15% accuracy.
The metric I used is accuracy
![image](https://user-images.githubusercontent.com/78225646/229761281-a38453bf-d789-4cee-9a95-6943c65620ec.png)
where
![image](https://user-images.githubusercontent.com/78225646/229761330-c9770728-ed26-4b4b-b842-adaf1ffd144d.png)
